openwisp_users/multitenancy.py

```diff
@@ -67,15 +94,12 @@ def _edit_form(self, request, form):
         fields = form.base_fields
         user = request.user
         org_field = fields.get('organization')
-        if user.is_superuser and org_field and not org_field.required:
-            org_field.empty_label = _('Shared systemwide (no organization)')
```

Comment:
Do we really want to remove this feature, would we need to give backward compatibility for it?

Reply:
I have not removed this. It is being used in `_get_empty_level` of class `SelectOrgMixin`